src/components/WalletFace/Actions/walletFaceActions.m.scss

```diff
@@ -0,0 +1,57 @@
+@import 'styles/_base';
+@import 'styles/mixins';
+@import 'styles/colors';
+
+.overlay {
+  z-index: 2;
```

Comment:
As the `z-index` is "global" attribute (nodes with it affect other nodes) better to control all its values in one place.
We have that one - styles/zindex. We should keep it up to date.